docs/mllib-statistics.md

```diff
@@ -335,8 +335,8 @@ featureTestResults.foreach { result =>
 
 <div data-lang="java" markdown="1">
 [`Statistics`](api/java/org/apache/spark/mllib/stat/Statistics.html) provides methods to 
-run Pearson's chi-squared tests. The following example demonstrates how to run and interpret 
-hypothesis tests.
+run Pearson's chi-squared tests. The following example demonstrates how to run a 1-sample
```

Comment:
Sorry, that line should not have been there. The anderson darling test is featured later (starting at line 424). Fixed in latest push.